common/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
```

Comment:
Just noting that some of these scripts are relying on the `set -eu` patch coming in #8693.